README.md

```diff
@@ -126,6 +126,44 @@ oc patch consoles.operator.openshift.io cluster \
   --patch '{ "spec": { "plugins": ["nmstate-console-plugin"] } }' --type=merge
 ```
 
+
+## Deployment on cluster with Helm
+
+A [Helm](https://helm.sh) chart is available to deploy the plugin to an OpenShift environment.
+
+
+Install the chart using the name of the plugin as the Helm release name into a new namespace or an existing namespace (openshift-namespace by default) parameter and providing the location of the image within the `image` parameter by using the following command:
+
+```shell
+helm upgrade -i  nmstate-console-plugin deployment/nmstate-console-plugin -n openshift-namespace --create-namespace
```

Comment:
the namespace here shouldn't be `openshift-nmstate`?

Reply:
right lol